app/routes.py

```diff
@@ -1,2 +1,304 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response, request
+from app import db
+import requests
+import os
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+tasks_bp = Blueprint("tasks", __name__, url_prefix = "/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
 
+
+# THE LINK TO TEST IN HEROKU IS https://task-list-project2.herokuapp.com/tasks
+
+
+@tasks_bp.route("",methods=["POST", "GET"])
```

Comment:
To practice the single responsibility principle, we can separate each request method into different functions.